contracts/Nexus.sol

```diff
@@ -100,14 +100,15 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
         uint256 missingAccountFunds
     ) external virtual payPrefund(missingAccountFunds) onlyEntryPoint returns (uint256 validationData) {
         address validator = op.nonce.getValidator();
-        if (!op.nonce.isModuleEnableMode()) {
+        if (op.nonce.isModuleEnableMode()) {
+            PackedUserOperation memory userOp = op;
+            userOp.signature = _enableMode(userOpHash, op.signature);
+            if (!_isValidatorInstalled(validator)) return VALIDATION_FAILED;
+            validationData = IValidator(validator).validateUserOp(userOp, userOpHash);
+        } else {
```

Comment:
Is there any reason the order changed? I think it's less likely to happen when calling with EnableMode enabled compared to when calling without EnableMode

Reply:
@filmakarov

Reply:
https://cantina.xyz/code/d1d4b139-9705-4367-9468-297b7078674e/findings/42